AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -124,35 +125,46 @@ def AtomEditorComponents_Light_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, light_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # 5. Cycle through all light types.
+        for ltype, value in LIGHT_TYPES.items():
+            light_component.set_component_property_value(
+                AtomComponentProperties.light('Light type'), LIGHT_TYPES[ltype])
+            current_light_type = light_component.get_component_property_value(
+                AtomComponentProperties.light('Light type'))
+            test_shape = (
```

Comment:
test_shape should be test_light_type maybe

Reply:
fixed